openmp/libomptarget/test/unified_shared_memory/close_modifier.c

```diff
@@ -5,6 +5,8 @@
 
 // amdgpu runtime crash
```

Comment:
Fails with: "PluginInterface" error: Failure to synchronize stream (nil): Error in cuStreamSynchronize: an illegal memory access was encountered